lib/handler/mruby.c

```diff
@@ -272,7 +272,7 @@ static mrb_value build_constants(mrb_state *mrb, const char *server_name, size_t
             mrb_ary_set(mrb, ary, i, lit);
         }
         for (; i != H2O_MAX_TOKENS * 2; ++i) {
-            const h2o_token_t *token = h2o__tokens + i - H2O_MAX_TOKENS;
+            const h2o_token_t *token = h2o__tokens + (i - H2O_MAX_TOKENS);
```

Comment:
This change is not related to what this PR wanted to fix, but undefined behavior sanitizer warns without this parenthesis because `h2o__tokens + i` overflows.